.neutrinorc.js

```diff
@@ -4,7 +4,22 @@ const jest = require('@neutrinojs/jest');
 
 module.exports = {
   use: [
-    reactLint(),
+    reactLint({
+      rules: {
+        'no-use-before-define': ["error", { 
+          /**
+           * Ignores reference to functions used before the function declaration. 
+           * Since function declarations are hoisted, this is considered safe.
+           */
+          "functions": false,
```

Comment:
We can probably remove the double quotations:

```
functions: false,
```

Reply:
Applied changes!